Added the Introduction to the Coding Interview Take Home Projects (#35581)
freeCodeCamp/freeCodeCamp: client/src/pages/learn/coding-interview-prep/take-home-projects/index.md

---
## Introduction to the Coding Interview Take Home Projects

> Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project which they can complete on their own time. This type of interviews usually requires a lot of work but it is considered to be the most preferred interviewing process.

Reviewer: ```suggestion
Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project which they can complete in their own time. These type of interviews usually require a lot of work but it is the most preferred interviewing process.
```

> Programming interviews have always been stressful. One of the solutions is giving the applicant a take home project to be completed outside of the interview. These types of interviews usually require a lot of work, but it is the most preferred interviewing process.

Reviewer 2: Most preferred interviewing process for whom?  The interviewer or interviewee?   Can you provide evidence for this? The previous verbiage was simpler to understand and meant something different.

Author: For both the Applicant and Technical HR @cmccormack

Reviewer 3: @Sumer16 As @cmccormack mentioned, we need a reference linked which states this - 

> but it is the most preferred interviewing process.

The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.

In this section you get the chance to work on many Take Home Projects like:

> *	Show Local Weather

Reviewer: Change this from tab indentation to 2 spaces

> The interview after project submission could be a discussion about applicant’s design decisions, scalability, and what they might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects to talk about.

Reviewer 4: It should be `the applicant's design decisions,`  Again, you can not use the word `they` to refer to a single applicant, so `they` should not be used here.  Also, you never end a sentence with a preposition (i.e `about`).  Lastely, "out of the box" is an expression which may confused readers not familiar with such expressions.  Consider rephrasing it in "plain" English.  I don't have a particular suggestion off the top of my head, but I am confident you will think of something.  Note that my suggestion below does not add an alternative for you, but does correct grammar and spelling mistakes.
```suggestion
The interview after the project submission could be a discussion about the applicant’s design decisions, scalability, and what the applicant might have done out of the box. This approach is more effective when the applicant doesn’t have a lot of publicly available past projects.
```

Author: I'm sorry for my bad grammar! Thank You for the suggestions. @RandellDawson

Reviewer 4: Looks good now.

*  Build a Wikipedia Viewer
*  Build a Recipe Box and many more...

> By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and prepare you for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.

Reviewer: We'll need to discuss if we want to list the umber of projects over here.

cc: @RandellDawson / @moT01

Reviewer 5: It says there's 20 projects - I don't think we need to say that twice.

Reviewer: I want to get the number removed completely. See https://github.com/freeCodeCamp/freeCodeCamp/pull/34405#discussion_r264282065

Reviewer: Same for here. Reword it to not say no of challenges

Author: So what changes needs to be done? @thecodingaviator

Reviewer: Reword it to remove no of challenges. Wait till the PR before this is merged

Author: I have changed it @thecodingaviator

> By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and make you prepared for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.

Reviewer: ```suggestion
By the end of this section, you will have 20 Take Home Projects under your belt. This will boost your confidence and prepare you for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.
```

> *  Build a Recipe Box and many more...

Reviewer 2: Each bullet could be punched up a bit, maybe something like

> * Show Local Weather using location and weather APIs

and many more... can either be removed, or maybe moved to the next line outside the bullets like
> and many more!

> By the end of this section, you will have plenty of Take Home Projects under your belt. This will boost your confidence and prepare you for Take Home Project Interviews. Have fun and remember to use the [Read-Search-Ask](https://www.freecodecamp.org/forum/t/how-to-get-help-when-you-are-stuck/19514) method if you get stuck.

Reviewer 2: I believe `take-home` with the hyphen is the proper way to write it.

Author: I think if we keep it similar to the section heading like "Take Home" it would look better than the hyphen.

Reviewer 2: Fix the header instead, then it will all be accurate.
